config/install/field.storage.node.field_skin_color.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+_core:
+  default_config_hash: msB96rMaXO5OwbjOBacIak6RrY3sHDRR1Fem_490Zrs
```

Comment:
Remove these.

Reply:
Done. All changes pushed and committed for review.